tasks/recover/print_info.yml

```diff
@@ -0,0 +1,17 @@
+- name: Print info about VMs that were registered
+  debug:
+    msg: "The following VMs were successfully registered: {{ succeed_vm_names }}"
```

Comment:
Suggestion: maybe it could be more readable like this:
  The following VMs registered successfully: ...